### PR TITLE
Re sync things

### PR DESCRIPTION
**Pull request by mmcc**
I originally was going to try and add some default timeouts to handlers, but that turned into a gigantic rabbit hole, so bailing on that for now.

Big thing this PR adds is the ability for sync to pick back up on any existing assets that aren't finished and finish the drill on them.

### CODE REVIEW

File: src/handlers/local-upload.ts [hunk 1]
```diff
@@ -101,14 +101,25 @@ async function pollForUploadAsset(filePath: string, asset: Asset) {
...
 
+  if (asset.status === 'ready') {
+    return;
+  } else if (asset.status === 'processing') {
+    console.log(`Asset is already processing. Polling for completion: ${asset.originalFilePath}`);
+    return pollForAssetReady(asset.originalFilePath, asset);
+  } else if (asset.status === 'uploading') {
+    // Right now this re-starts the upload from the beginning.
+    // We should probably do something smarter here.
+    console.log('Resuming upload...');
```
Comment (luwes) [inline src/handlers/local-upload.ts:120]: 👍

File: package.json [hunk 2]
```diff
@@ -50,7 +51,9 @@
     "@types/node": "^20.4.5",
     "@types/react": "^18.2.16",
     "@types/yargs": "^17.0.24",
+    "glob": "^10.3.3",
```
Comment (luwes) [inline package.json:54]: did you end up using glob?

**Review by luwes — APPROVED** [review]
nice! lgtm

File: package.json (hunk 2 above)
Comment (mmcc) [reply package.json:54]: Good catch, I did not!

**Pull request merged**